vllm/utils.py

```diff
@@ -628,14 +628,34 @@ def is_valid_ipv6_address(address: str) -> bool:
         return False
 
 
+def split_host_port(host_port: str) -> Tuple[str, int]:
+    # ipv6
+    if host_port.startswith('['):
+        host, port = host_port.rsplit(']', 1)
+        host = host[1:]
+        port = port.split(':')[1]
+        return host, int(port)
+    else:
+        host, port = host_port.split(':')
+        return host, int(port)
```

Comment:
This could be slightly improved to raise `ValueError` directly with a message explaining that the string is an invalid combination of host + port. Right now it'll be something less clear, like `ValueError: too many values to unpack (expected 2)`, depending on how it was formatted